uint8 front_left
uint8 front_right
uint8 back_left
uint8 back_right
